### ISSUE TITLE
Delete Dimension

### ISSUE THREAD

**Issue by ahmedrizwan239** [theoephraim/node-google-spreadsheet#649]
I was trying to delete some rows, but it didn't work for me, on spectating the function in the core file i.e GoogleSpreadsheetWorksheet.js I found out that is a void function, I would like contribute to make this function workable.

**Comment by theoephraim** [comment]
yeah you can delete populated rows via the row based api, but using the deleteDimension call is not yet implemented. Feel free to open a PR - it should be fairly straightforward. Otherwise I'll try to get to it when I can!

**Comment by ahmedrizwan239** [comment]
Thanks for the opportunity,  I've opened a pull request. Do have a look, Thanks!
